app/src/main/java/com/nervousfish/nervousfish/activities/VisualVerificationActivity.java

```diff
@@ -70,4 +93,38 @@ public void buttonAction(final View v) {
         }
     }
 
+    @Override
+    protected void onStart() {
+        super.onStart();
+        this.serviceLocator.registerToEventBus(this);
+    }
+
+    @Override
+    protected void onStop() {
```

Comment:
Why doesn't checkstyle trigger here so we don't have to worry about smaal things like this?